src/app/components/activity-card/activity-card.component.html

```diff
@@ -1,6 +1,49 @@
-<ion-card color="primary" class="card practera-card">
-  <ion-card-content padding>
-    <h4 color="primary-contrast">Activity</h4>
-    <h1 color="primary-contrast">{{name}}</h1>
+<ion-card 
+*ngIf="activity; else reviewCard"
```

Comment:
The reason of re-using a component is that we wanna use the same styling for different places. If we totally separate those layouts for `activity` and `review`, then there's no point that we use the same component.
That means we should 
1. Keep styling the same for main elements on the card.
2. Use a toggle to decide whether show this element or not.

Reply:
We can have a broader discussion with more people about which elements should be in the same styling. Please arrange that meeting if you think is necessary